components/slug/TableRender.tsx

```diff
@@ -0,0 +1,30 @@
+import useWindowDimensions from '../../hooks/useWindowDimensions';
+import PartnersBlocks from './PartnersBlocks';
+import PartnersTable from './PartnersTable';
+import AdminsBlocks from './AdminsBlocks';
+import AdminsTable from './AdminsTable';
+import { ISlug } from '../../types/slug';
+
+
+
+function TableRender({ data }: ISlug) {
+  const { width } = useWindowDimensions();
+
+  return (
+    <div>
+      {width > 710 ? (
```

Comment:
<img width="166" alt="image" src="https://user-images.githubusercontent.com/16991899/189646029-57a86101-6f0b-4400-ac57-d5d1d1582165.png">
I think it's better to find a common value and create a hook that will return isMobile true or false